tests/standalone_tests/test_udfgenerator.py

```diff
@@ -1338,7 +1347,82 @@ def expected_udf_output_tables(self):
             {
                 "tablename_placeholder": "main_output_table_name",
                 "drop_query": "DROP TABLE IF EXISTS $main_output_table_name;",
-                "create_query": "CREATE TABLE $main_output_table_name(node_id VARCHAR(500),dim0 INT,dim1 INT,val REAL);",
+                "create_query": 'CREATE TABLE $main_output_table_name("node_id" VARCHAR(500),"dim0" INT,"dim1" INT,"val" REAL);',
+            }
+        ]
+
+
+class TestUDFGen_TensorParameterWithCapitalLetter(
```

Comment:
This test is needed to test the behavior of MonetDB with respect uppercase characters. I think we should have an integration test as well to confirm the expected behavior (using the `use_database` fixture).

Reply:
Fixed